fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneOnlineIndexingTest.java

```diff
@@ -760,6 +780,43 @@ void luceneOnlineIndexingTestGroupingKeysBackgroundMerge(int groupingCount) thro
         assertTrue(newLength < oldLength);
     }
 
+    @ParameterizedTest
+    @BooleanSource
+    void luceneOnlineIndexingTestNoMergeIfFilteredOutRecords(boolean filterOut) throws IOException {
```

Comment:
`LuceneOnlineIndexingTest` does not seem like the best file for this, given that it is not about online indexing. Perhaps `LuceneIndexMaintenanceTest`